### PR TITLE
feat(i18n): add Japanese (ja) locale support

### PR DESCRIPTION
**Pull request by amine-benselim**

### CODE REVIEW

**Comment by amine-benselim** [comment]
UI-kit updated so this can be reviewed now 🙏

File: packages/application-shell/src/components/project-suspended/project-suspended.spec.js [hunk 1]
```diff
@@ -97,8 +97,10 @@ describe('rendering', () => {
         ],
       }
     );
-    await waitForElement(() =>
-      getByText(/Your Project is temporarily suspended due to maintenance/)
+    await wait(() =>
+      expect(
+        queryByText(/Your Project is temporarily suspended due to maintenance/)
+      ).toBeInTheDocument()
```
Comment (emmenko) [inline packages/application-shell/src/components/project-suspended/project-suspended.spec.js]: Hmm isn't this the same thing as before?

Comment (amine-benselim) [reply packages/application-shell/src/components/project-suspended/project-suspended.spec.js]: I found this to be weird too. the other way it was throwing a warning. this way it did not 🤷‍♂

**Comment by amine-benselim** [comment]
@emmenko can you please check Percy changes 🙏

**Comment by emmenko** [comment]
I retriggered the VRTs...please have a look at them as there might be some errors

**Comment by amine-benselim** [comment]
@emmenko I have no idea what to look for. the build failed and it says `Missing resources`

**Comment by emmenko** [comment]
Check the e2e percy snapshots: the icons for the links are missing

**Pull request merged**